### PR TITLE
add bash or python to index

### PR DESCRIPTION
**Pull request by jdkandersson**

### CODE REVIEW

**Review by weiiwang01 — DISMISSED** [review]

**Review by arturo-seijas — DISMISSED** [review]
LGTM

**Review by merkata — DISMISSED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 - [Charm Configuration Option Description](#charm-configuration-option-description)
 - [Charm Ubuntu and Python Version](#charm-ubuntu-and-python-version)
 - [Dependencies](#dependencies)
+- [When to use Python or Shell](#when-to-use-python-or-shell)
```
Comment (amandahla) [inline CONTRIBUTING.md:6]: Would be ok to change for keeping the index in alphabetical order?

Comment (jdkandersson) [reply CONTRIBUTING.md:6]: Ah thats the sorting algorithm, I had been trying to figure it out, I got lucky last time when adding dependencies

**Review by arturo-seijas — APPROVED** [review]
LGTM

**Review by gregory-schiano — APPROVED** [review]
LGTM

**Pull request merged**